model/cpython/registry.yaml

```diff
@@ -0,0 +1,25 @@
+groups:
+  - id: registry.cpython
+    type: attribute_group
+    display_name: CPython attributes
+    brief: >
+      This document defines CPython related attributes.
+    attributes:
+      - id: cpython.gc.generation
+        stability: development
+        type:
+          members:
+            - id: gen0
+              value: 'gen0'
+              brief: "Generation 0"
+              stability: development
+            - id: gen1
+              value: 'gen1'
+              brief: "Generation 1"
+              stability: development
+            - id: gen2
+              value: 'gen2'
+              brief: "Generation 2"
+              stability: development
```

Comment:
@xrmx one quick question: is there specific rationale behind the enum `gen0`, `gen1`, `gen2` vs just an integer?

Reply:
No specific reason

Reply:
are `gen0`, `gen1`, `gen2` common names for these generations in cpython? I didn't have much too luck in google: https://www.google.com/search?q=%22gen0%22+%22gen1%22+%22gen2%22+cpython

Reply:
@trask Not really, I chose these values because other counters like threshold and count get the X suffix https://docs.python.org/3/library/gc.html in other apis. Will switch to integer.

Reply:
Done